about.html

```diff
@@ -0,0 +1,154 @@
+                The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two. This year, we hope to expand our invitation list including organizations and individuals.
+            </p></div>
+            <div class="highlevel-section-div mt-4 mb-4" style="text-align: center;"><p style="font-weight: 700; font-size: 1rem; color: #272a31;">
+                Please contact us per Email for any further questions about CC Global Summit 2015!<br>
+                <b style="text-align: center; text-decoration: underline; font-weight: 700; font-size: 1rem; color: #272a31;">ccsumit2015@cckorea.org</b>
+            </p>
+        </div>
+        </main>
+        <section id="mainprogram-section" class="highlevel-section pt-5 pb-4">
+            <div class="highlevel-section-div" style="display: flex; flex-direction: column; color: #272a31; font-size: 1.5rem; font-weight: 700;">
+                <p style="text-align: center; ">Creative Commons<br>Global Summit 2015 Logo</p><hr style="border: 1px solid #ec5242; width: 10%; margin-top: 0%;">
+                </div>
+    
+                <div class="highlevel-section-div mt-2 mb-4">
+                    <p class="p-2" style="text-align: center; color: #272a31; font-size: 0.9rem; font-weight: 700;">The logo of CC Global Summit 2015 was decided through the logo competition from 8.June to 7.July.</p>
+                    </div>
+                <div class="highlevel-section-div" style="border: 1px solid #d3d3d3"><img src="./assets/LogoTWOH.PNG" alt="The Way of Humanity" />
+                    
+                </div>
+        </section>
+        <hr style="border: 1px solid #d3d3d3; width: 100%">
+        <section id="featuredspeakers-section" class="highlevel-section">
+            <div class="highlevel-section-div mb-2" style="display: flex; flex-direction: column; color: #272a31; font-size: 1.5rem;">
+                <p style="font-weight: 700;">See the past CC Global Summits</p><hr style="border: 1px solid #ec5242; width: 10%; margin-top: 0%;">
+                </div>
+                
+                <div class="highlevel-section-div mb-5">
+                    <p style="text-align: center; font-weight: 700;">Take a look at the last two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+                    </div>
+                <div class="highlevel-section-div p-1" style="flex-direction: column;">
+                    <div id="card-1" class="highlevel-section-div-2 mb-4">
+                        <div style="text-align: center;"><p>A</p><br><p>CC Global Summit 2013 in Buenos Aires</p></div>
+                    </div>
+                    <div id="card-2" class="highlevel-section-div-2">
+                        <div style="text-align: center;"><p>B</p><br><p>CC Global Summit 2011 in Warsaw</p></div>
+                    </div>
+                </div>
+        </section>
```

Comment:
- [ ] Please align the images horizontally and make sure both images are the same size. Also make sure the font-weight of the paragraph "Take a look at the last two CC Global Summits...." is lighter. Currently your design looks bolder compared to the original design.

| Cindy Shin design    | Your design |
| ------------- | ------------- |
|<img width="576" alt="Screenshot 2023-04-21 at 12 11 32 PM" src="https://user-images.githubusercontent.com/44798044/233578701-17675e86-2c4d-4759-82c0-8c6d3549ea21.png">| <img width="360" alt="Screenshot 2023-04-21 at 11 33 33 AM" src="https://user-images.githubusercontent.com/44798044/233578795-b878ee8a-663e-48f6-b66f-93297c003602.png">|